README.md

```diff
@@ -13,6 +13,33 @@ Import on SBT:
 
     "com.codacy" %% "bitbucket-scala-client" % "<VERSION>"
 
+To build the `BitbucketClient` you need to pass a builder function from Unit => WSClient
+
+#### Play 2.4
+
+```Scala
+import com.ning.http.client.AsyncHttpClientConfig
+import play.api.libs.ws.ning.{NingAsyncHttpClientConfigBuilder, NingWSClient}
+
+() => {
+  val config = new NingAsyncHttpClientConfigBuilder().build()
+  val clientConfig = new AsyncHttpClientConfig.Builder(config).build()
+  val client = new NingWSClient(clientConfig)
+}
+```
+
+#### Play 2.7
+
+```Scala
+import akka.stream.Materializer
+import play.api.libs.ws.ahc.AhcWSClient
```

Comment:
I'm trying to do this in remote-provider-service that is using play 2.7 and I can't access this class. Do I need to import any extra dependency to have access to this?

Reply:
yes `play-ahc-ws`, but let speak a second about this ...

Reply:
@machadoit @peavers @prendi 
Talked with @andreaTP about this and we can do 3 things regarding this:
1. Add the new dependency `play-ahc-ws` on RPS which will be a bit weird since it is kind of a play "wrapper" over the AsyncHttpClient that RPS already has.
2. Try to abstract the http client that we are using in the library.
3. Do what was done in the stash-scala-library and start using scalaj-http.

What should we do?

Reply:
Can we do the least we need to move this task fwd? We shouldn't be doing detours to deliver a feature, but if a way is clearly better, let's create a task and tackle it on its own

Reply:
Option one seems the quickest to progress, I would really hate to see us loose a week or more changing the client over to `scalaj-http` as it looks a fairly involved based on the commits on the stash-client.

If we went with the first option but had a tech task to switch out the client when we have more time?

Reply:
We ended up receiving only the materializer for Play 2.7 builds and build the client inside. For Play 2.4 (like portal) it will be used the same way

Reply:
Two things to say:
 - is totally fine to proceed with option 1 having a task to remember to come back to this at some point
 - switching to `scalaj` should take much, much less than a week :-)

Reply:
Not sure if I understand your bullets, are you fine with the solution as it is? We ended up not needing to add the dependency on RPS side, because we build the client inside with the ahc-standalone. And we only need to pass the Materializer when created from Play 2.7 lib